frontend/app/src/components/accounts/AccountTopTokens.vue

```diff
@@ -1,19 +1,31 @@
 <script lang="ts" setup>
 import type { AssetBalance } from '@rotki/common';
+import type { BlockchainAccountGroupWithBalance, BlockchainAccountWithBalance } from '@/types/blockchain/accounts';
 
 const props = defineProps<{
   chains: string[];
-  address: string;
+  row: BlockchainAccountWithBalance | BlockchainAccountGroupWithBalance;
```

Comment:
that would be https://github.com/rotki/rotki/blob/6c565534a7dff68fed579a412a46cea03488ecc3/frontend/app/src/types/blockchain/accounts.ts#L84 correct?

Reply:
yes I think you're right